velox/docs/functions/spark/aggregate.rst

```diff
@@ -107,10 +107,16 @@ General Aggregate Functions
 
     Returns the sum of `x`.
 
-    Supported types are TINYINT, SMALLINT, INTEGER, BIGINT, REAL and DOUBLE.
+    Supported types are TINYINT, SMALLINT, INTEGER, BIGINT, REAL, DOUBLE and DECIMAL.
 
     When x is of type DOUBLE, the result type is DOUBLE.
     When x is of type REAL, the result type is REAL.
+
+    When x is of type DECIMAL(p, s), the result type is DECIMAL(p + 10, s), and p + 10 will not exceed 38.
```

Comment:
Nice comment.

> and p + 10 will not exceed 38

It is not possible to force p + 10 to not exceed 38. Maybe change to ", where (p + 10) is capped at 38.